sdk/schemaregistry/schema-registry/src/constants.ts

```diff
@@ -1,4 +1,12 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+/**
+ * @internal
+ */
 export const DEFAULT_SCOPE = "https://eventhubs.azure.net/.default";
+
+/**
+ * @internal
+ */
+export const SDK_VERSION = "1.0.2";
```

Comment:
The only thing that I would recommend is adding this to the `//constantPaths` section of package.json so that the sdk bot can bump the version automatically 👍